### DISCUSSION TITLE
Unable to serve the cloned repository on local host so as to make changes

### DISCUSSION THREAD

**Discussion by sunday-ucheawaji** [dowjones/react-dropdown-tree-select#633] [General]
**Describe the bug**
I cloned the repository, installed the library and then did `npm run demo` to watch any change made but I get this on localhost "http://[::]:3000/". If i go to localhost:3000, I get another error, this error
Not Found / 404
You may be seeing this error due to misconfigured options.
Please see [this FAQ question](https://github.com/shellscape/webpack-plugin-serve/blob/master/.github/FAQ.md#what-does-the-not-found--404-error-mean) for information on possible causes and remedies.

I get this in the browser console
VM117:5 crbug/1173575, non-JS module files deprecated.

Static Paths
C:\Users\sunda\Documents\react-dropdown-tree-select

**To Reproduce**
git clone library
npm install
npm run demo

**Environment (please complete the following information):**
 - Browser [e.g. chrome v60.12.0, safari 10.1]
 - OS: [e.g. iOS, Windows x64] (if applicable)
 - node version (if applicable)
 - yarn or npm version (if applicable)

**Additional context**

**Comment by mrchief** [comment]
I'm not sure what's going on with your local setup but you can try passing `--host` to webpack-serve
